cli/create-scss.js

```diff
@@ -34,6 +34,7 @@ var writeScss = function(iconInfos, scssPath) {
 		var iconInfo = iconInfos[i];
 
 		scss.write('@mixin ' + iconInfo.mixin + '() {\n');
+		scss.write('	@include d2l-icon();\n');
```

Comment:
There'll definitely be some duplication for sure -- but we're now NOT including this CSS in BSI, so it would just be within people's apps. And hopefully apps aren't using more than a few icons each?